content/applications/general/email_communication/email_domain.rst

```diff
@@ -1,37 +1,27 @@
-=======================
-Send an email with Odoo
-=======================
+=========================================
+Configuring DNS Records for Email Sending
+=========================================
 
-Use an email domain in Odoo
-===========================
+SPAM Email
+----------
 
-Documents in Odoo (such as a CRM opportunity, a sales order, an invoice, etc.) have a discussion
-thread, called *chatter*.
+It happens sometimes that some emails from Odoo are misclassified by the different email providers
+and end up in spam folders. At the moment, some settings are out of Odoo's control, notably the
+way the different email providers classify our emails according to their own restriction policy
+and/or limitations.
 
-When a database user posts a message in the chatter, this message is sent by email to the followers
-of the document. If a follower replies to the message, the reply updates the chatter, and Odoo
-relays the reply to the followers as a notification.
+In order for servers to accept emails from Odoo on a more regular basis, one of the solutions is
+for customers to create rules within their own mailbox. A filter can be added to the email inbox so
+that when email is received from the catchall (catchall@yourdatabase.odoo.com) it is moved to the
+inbox. It is also possible to add the Odoo database domain onto a safe senders list or whitelist
+on the receiving domain.
 
-Messages sent in the chatter from internal database users to external users (such as partners,
-customers, or vendors) are relayed on behalf of the database users. Messages sent back to the
-chatter from external users will appear in the chatter from their respective email addresses, or as
-they are listed in their Contacts record.
+If you notice a Odoo mail server on a blacklist notify Odoo via a new help ticket and the
+technical support team will work to get the servers removed from the blacklist.
 
-If the Odoo database is hosted on the cloud (Odoo Online or Odoo.sh), it is not necessary to add an
-outgoing email server to send emails from a custom domain.
-
-.. important::
-   The Odoo server is subject to a daily email limit to prevent abuse. The default limit is 200
-   emails sent per day for databases with an **Enterprise** subscription. This limit can be
-   increased under certain conditions. See the :doc:`FAQ <faq>` or contact support for more
-   information.
-
-To ensure that emails sent to and from the chatter reach their intended contacts, instead of being
-considered spam, Odoo recommends configuring the domain name.
-
-For the same reason, Odoo also recommends giving each database user an email address from the
-configured domain, rather than a generic email address domain (such as gmail.com, outlook.com,
-etc.).
+Should the Odoo database be using a custom domain for sending emails from Odoo there are three
+records that should be implemented on the custom domain's DNS to ensure the sending and receiving
```

Comment:
SPF DKIM & DMARC are only impacting the deliverability, not the receiving part.

Also as we were again discussing today with @jad-odoo , unfortunately, there is use case where even with a full and correct DNS configuration, we are unable to guarantee that emails do not arrive in SPAM. As the final receiving mailbox configuration might decide something different.

That's the reason why we can see emails arriving in Spam in Outlook, even with DNS fully and correctly configured but everything working smoothly in Gmail.